docs/csharp/toc.yml

```diff
@@ -1344,12 +1344,8 @@ items:
       items:
       - name: Pattern matching
         href: ../../_csharplang/proposals/csharp-7.0/pattern-matching.md
-      - name: Local functions
```

Comment:
Was read-only structs not in the TOC?

Reply:
It wasn't. That spec was a placeholder that didn't have any content.